vignettes/canham.Rmd

```diff
@@ -94,10 +87,12 @@ Tree_Size_Ests <- hmde_extract_estimates(model = "canham_multi_ind",
                                  fit = tree_canham_fit,
                                  input_measurement_data = Tree_Size_Data)
 
-saveRDS(tree_canham_fit, "inst/extdata/tree_canham_fit.rds")
-saveRDS(Tree_Size_Ests, "inst/extdata/Tree_Size_Ests.rds")
+# saveRDS(tree_canham_fit, "tree_canham_fit.rds")
+# saveRDS(tree_canham_estimates, "tree_canham_estimates.rds")
 ```
 
+We have saved the model output for you so you can follow along this case study without running the model.
```

Comment:
I think we can remove this as you already mentioned users can access the estimates on L79

Reply:
Done.